app/ab_test/a/templates/Donation_Confirmation_Phone.html.twig

```diff
@@ -116,7 +116,7 @@
 						</div>
 
 						<div class="payment f-left">
-							{% if donation.paymentType == 'PPL' or donation.paymentType == 'MCP' %}
+							{% if donation.paymentType == 'PPL' or donation.paymentType == 'MCP' or donation.paymentType == 'SUB' %}
```

Comment:
After some discussion w/ @KaiNissen and @gbirke decided to leave it like that until further notice (stakeholder).